### PR TITLE
Taxi feed: headers bug

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-25555

## Description
Fixed an issue where the authentication by a custom header was not worked properly.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ApiModules/ReleaseNotes/2_2_19.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### TAXII2ApiModule
+Fixed an issue where the authentication by a custom header was not work properly.
```
Comment (ShahafBenYakir) [inline Packs/ApiModules/ReleaseNotes/2_2_19.md:4]:
```suggestion
Fixed an issue where the authentication by a custom header was not working properly.
```

File: Packs/FeedDHS/ReleaseNotes/2_0_15.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### DHS Feed v2
+
+Fixed an issue in the **TAXII2ApiModule** script.
```
Comment (ShahafBenYakir) [inline Packs/FeedDHS/ReleaseNotes/2_0_15.md:6]: Is this using the TAXII2ApiModule? Does it use the authentication headers?

File: Packs/FeedMitreAttackv2/ReleaseNotes/1_1_17.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### MITRE ATT&CK
+
+Fixed an issue in the **TAXII2ApiModule** script.
```
Comment (ShahafBenYakir) [inline Packs/FeedMitreAttackv2/ReleaseNotes/1_1_17.md:6]: Is this using the TAXII2ApiModule? Does it use the authentication headers?

File: Packs/FeedTAXII/ReleaseNotes/1_1_30.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### TAXII 2 Feed
+
+Fixed an issue where the authentication by a custom header was not work properly.
```
Comment (ShahafBenYakir) [inline Packs/FeedTAXII/ReleaseNotes/1_1_30.md:6]: see the API module rn comment

**Review by ShahafBenYakir — COMMENTED** [review]
Attach the issue it is fixing

File: Packs/FeedDHS/ReleaseNotes/2_0_15.md (hunk 2 above)
Comment (moishce) [reply Packs/FeedDHS/ReleaseNotes/2_0_15.md:6]: no

File: Packs/FeedMitreAttackv2/ReleaseNotes/1_1_17.md (hunk 3 above)
Comment (moishce) [reply Packs/FeedMitreAttackv2/ReleaseNotes/1_1_17.md:6]: no

**Review by ShahafBenYakir — APPROVED** [review]
See my other comment

**Review by Ni-Knight — APPROVED** [review]
Looks good.

**Pull request merged**